tests/time_tests/test_runner/test_config.yml

```diff
@@ -0,0 +1,8 @@
+- device:
+    name: "CPU"
+  model:
+    path: "/nfs/inn/proj/vdp/vdp_tests/stress_tests/master_04d6f112132f92cab563ae7655747e0359687dc9/caffe/FP32/alexnet/alexnet.xml" # TODO: add link to `test_data` repo model
```

Comment:
I thought we had a way to eliminate internal infrastructure paths? Also running those paths on windows is impossible.

Reply:
Added `expand_env_vars` and converted path to share with env variable. New version is supported by Win too